unlock-app/src/components/content/CheckoutContent.tsx

```diff
@@ -30,12 +55,25 @@ export const CheckoutContent = ({ account, network }: CheckoutContentProps) => {
       {account && (
         <BrowserOnly>
           <Account network={network} account={account} />
+          {loading && <Loading />}
+          {locks && (
+            <div>
+              {locks.map(lock => (
+                <Lock
+                  name={lock.name}
+                  keysAvailable={lockKeysAvailable(lock)}
+                  key={lock.name}
+                  price={lock.keyPrice}
+                  symbol={(lock as any).currencySymbol || 'ETH'}
```

Comment:
Right now this code assumes that any lock that does not have a currency symbol in the payload is an ETH lock. That may revisiting in the future.